resources/templates/Resource.json

```diff
@@ -0,0 +1,57 @@
+{
+    "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Resources event with additional metadata for generating mock values.",
```

Comment:
nit: Usage of "EventGridEvent" vs "Event Grid event" is inconsistent in these jsons